src/tv2-common/migrations/graphic-defaults.ts

```diff
@@ -1,6 +1,15 @@
-import { TV2ShowstyleBlueprintConfigBase } from '../blueprintConfig'
+export interface PriorDefaultTableConfigItemGfxTemplates {
+	VizTemplate: string
+	SourceLayer: string
+	LayerMapping: string
+	INewsCode: string
+	INewsName: string
+	VizDestination: string
+	OutType: string
+	IsDesign: boolean
+}
 
-export const DEFAULT_GRAPHICS: TV2ShowstyleBlueprintConfigBase['GFXTemplates'] = [
+export const DEFAULT_GRAPHICS: PriorDefaultTableConfigItemGfxTemplates[] = [
```

Comment:
Let this be a continuation of this discussion https://github.com/tv2/sofie-blueprints-inews/pull/149#discussion_r983320148 I'm not a big fan of keeping this defaults in their old _schema_/_structure_(?), but I also can see how this might make sense. @LindvedKrvang, you had some good thoughts about migrations, so what would you prefer: keeping the default values as they were, and letting the migrations do the job of moving entries around and removing the properties, versus using the new type here, splitting the defaults, and just forgetting `IsDesign` existed (except in the actual migrations that check if this property is still present)?

Reply:
I think this comes down to which philosophy we want for our migrations.

One school of thought is that a migration is a snapshot of a specific point in time and we want to migrate that specific point in time to another specific point in time. I.e. we have a table that is missing a column so we want a migration that adds said column. Whenever our migration is rerun, it will always be rerun from a state where said column is missing. Any migration written after will be run from a 'snapshot' where the column now already exists. 
In this school of though, whenever a migration has been run and sent to production it can not be edited. If you want to change something about the column, for example to remove it, you write a new migration.

The school of though described above is (as far as I know) the most common way of how migrations work. Unfortunately, this is not how our migrations in Sofie works.

The idea about having 'default values' that we will then continuesly update doesn't really fit into this way of thinking (atleast not our current implementation of it). I'm open for suggestions and think we should have a talk about how we want our default values.

But to answer your question. I lean against not using the new type here. The reason is that we would then be coupling our migration to the current implementation of the source code. When the source code changes (which it will. That is what happened here) then our migration would have to change as well, which in my opinion is something a migration should never have to do. 

I think we should have talk in the team about how we want our migrations to work and what changes we need to make (if any) to realize what we want.